src/cubi_tk/sodar/ingest_data.py

```diff
@@ -431,6 +415,9 @@ def build_jobs(self, hash_ending) -> tuple[str, tuple[TransferJob, ...]]:  # noq
                             collection_name=collection_name,
                             **match_wildcards,
                         )
+                        #if onko and germline analysisdata change analysis to germline_analysis
+                        if self.args.preset == "onk_analysis" and "DragenGermline" in path:
+                            remote_file.replace("analysis", "germline_analysis")
```

Comment:
Maybe instead of hard coding this, we should consider adding an option to do string/regex replacement in the remote filenames/paths (analogous to the current -m which acts on sample name only). Then we could add "inbuilt" defaults for these to the presets. 
Doesn't have to be done right away though I guess